src/bin/stratisd.rs

```diff
@@ -75,15 +69,21 @@ fn main() {
         .get_matches();
 
     if matches.is_present("debug") {
-        unsafe { DEBUG = true }
-    };
+        let mut builder = LogBuilder::new();
```

Comment:
Maybe:

```
let mut builder = LogBuilder::new();
if matches.is_present("debug") {
    builder.filter(Option("stratisd"), LogLevelFilter::Debug);
    builder.filter(Option("libstratis"), LogLevelFilter::Debug);
} else {
    if let Ok(s) = env::var("RUST_LOG") {
        builder.parse(&s);
    }
};
builder.init().unwrap();
```

Then it is very clear that the intention is that ```--debug``` flag overrides whatever might be in the environment variable.

Regarding that doc sentence, to me it's meaning is quite uncertain. "on a per-module basis" is just a terminally vague phrase.